libgnucash/engine/gnc-pricedb.h

```diff
@@ -573,19 +571,27 @@ gnc_numeric gnc_pricedb_get_nearest_before_price (GNCPriceDB *pdb,
                                                   const gnc_commodity *new_currency,
                                                   const time64 t);
 
-/** @brief Retrieve the price one currency to another at specified date
+/** @brief Retrieve the price one currency to another using the price
```

Comment:
That is doxygen and generates our api docs..
Docbook is used as source language in gnucash-docs.
;-)